go/pkg/pwapi/api_challenge-subscription-validate.go

```diff
@@ -171,6 +171,18 @@ func (svc *service) ChallengeSubscriptionValidate(ctx context.Context, in *Chall
 			return errcode.ErrAgentUpdateState.Wrap(err)
 		}
 
+		// get nb of validations
+		var nb int64
+		svc.db.
+			Table("season_challenge").
+			Joins("inner join challenge_subscription cs on season_challenge.id = cs.season_challenge_id").
+			Joins("inner join challenge_validation cv on cs.id = cv.challenge_subscription_id").
+			Where(pwdb.SeasonChallenge{ID: subscription.SeasonChallengeID}).
+			Count(&nb)
+
+		// compute score : 1 / (x/10 + 1) * 95 + 5
+		validationScore := 1/(nb/10+1)*95 + 5
+
```

Comment:
add a new query below that will set a flag for each team validating that challenge in the current season to mark them as "OutdatedScore".

The frontend will use this flag to add a visual indicator.
And use another script that will periodically update all the scores.

As soon as we'll be stable with this new standalone script. Then we'll try to call the script automatically on each event that should trigger a score update.